src/App.js

```diff
@@ -1,8 +1,25 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <h1>React sum</h1>
+    <Sum
+      a={2}
+      b={2}
+    />
```

Comment:
```suggestion
    <Sum a={2} b={2} />
```
меньше 3 пропсов - можно писать в 1 строку